setup.py

```diff
@@ -6,13 +8,33 @@ def readme():
         return f.read()
 
 
+def _is_old_setuptool():
+    """
+    Return True if setuptool is older than 20.5 which version support
```

Comment:
How about `... which introduced environment markers for requirements.`? The current sentence does not seem to be grammatically correct.

Reply:
Fixed. Thanks.